src/lib/parser.c

```diff
@@ -167,6 +167,19 @@ __Z_INLINE parser_error_t parser_printPublicKey(publickey_t *pk,
     return parser_ok;
 }
 
+__Z_INLINE parser_error_t parser_printSignature(raw_signature_t *s,
+                                                char *outVal, uint16_t outValLen,
+                                                uint8_t pageIdx, uint8_t *pageCount) {
+
+    // 64 * 2 + 1 (one more for the zero termination)
+    char outBuffer[129];
+    MEMZERO(outBuffer, sizeof(outBuffer));
+
+    array_to_hexstr(outBuffer, (const uint8_t *) s, 64);
```

Comment:
It would be better to have `sizeof(raw_signature_t)` instead of 64
similarly for 129 = 2 * sizeof(raw_signature_t) + 1